packages/SystemUI/src/com/android/systemui/biometrics/FingerprintDialogView.java

```diff
@@ -31,10 +34,18 @@
 public class FingerprintDialogView extends BiometricDialogView {
 
     private static final String TAG = "FingerprintDialogView";
+    private final boolean mHasFodSupport;
 
     public FingerprintDialogView(Context context,
             DialogViewCallback callback) {
         super(context, callback);
+        mHasFodSupport = FodUtils.hasFodSupport(context);
```

Comment:
why not use LineageContextConstants.Features.FOD ?
let's not add new overlays 
https://github.com/crdroidandroid/android_packages_apps_crDroidSettings/blob/10.0/src/com/crdroid/settings/fragments/LockScreen.java#L89

Reply:
OK